x-pack/plugins/security_solution/public/management/pages/host_isolation_exceptions/view/host_isolation_exceptions.tsx

```diff
@@ -0,0 +1,107 @@
+import { i18n } from '@kbn/i18n';
+import React, { useCallback } from 'react';
+import { EuiSpacer } from '@elastic/eui';
+import { FormattedMessage } from '@kbn/i18n/react';
+import { ExceptionItem } from '../../../../common/components/exceptions/viewer/exception_item';
+import {
+  getCurrentLocation,
+  getListFetchError,
+  getListIsLoading,
+  getListItems,
+  getListPagination,
+} from '../store/selector';
+import {
+  useHostIsolationExceptionsNavigateCallback,
+  useHostIsolationExceptionsSelector,
+} from './hooks';
+import { PaginatedContent, PaginatedContentProps } from '../../../components/paginated_content';
+import { Immutable } from '../../../../../common/endpoint/types';
+import { AdministrationListPage } from '../../../components/administration_list_page';
+import { SearchExceptions } from '../../../components/search_exceptions';
+import { ArtifactEntryCard, ArtifactEntryCardProps } from '../../../components/artifact_entry_card';
+import { HostIsolationExceptionsEmptyState } from './components/empty';
+
+type HostIsolationExceptionPaginatedContent = PaginatedContentProps<
+  Immutable<ExceptionListItemSchema>,
+  typeof ExceptionItem
+>;
+
+export const HostIsolationExceptions = () => {
+  const listItems = useHostIsolationExceptionsSelector(getListItems);
+  const pagination = useHostIsolationExceptionsSelector(getListPagination);
+  const isLoading = useHostIsolationExceptionsSelector(getListIsLoading);
+  const fetchError = useHostIsolationExceptionsSelector(getListFetchError);
+  const location = useHostIsolationExceptionsSelector(getCurrentLocation);
+
+  const navigateCallback = useHostIsolationExceptionsNavigateCallback();
+
+  const handleOnSearch = useCallback(
+    (query: string) => {
+      navigateCallback({ filter: query });
+    },
+    [navigateCallback]
+  );
+
+  const handleItemComponentProps = (element: ExceptionListItemSchema): ArtifactEntryCardProps => ({
+    item: element,
+    'data-test-subj': `hostIsolationExceptionsCard`,
+  });
+
+  const handlePaginatedContentChange: HostIsolationExceptionPaginatedContent['onChange'] =
+    useCallback(
+      ({ pageIndex, pageSize }) => {
+        navigateCallback({
+          page_index: pageIndex,
+          page_size: pageSize,
+        });
+      },
+      [navigateCallback]
+    );
+
+  return (
+    <AdministrationListPage
+      title={
+        <FormattedMessage
+          id="xpack.securitySolution.hostIsolationExceptions.list.pageTitle"
+          defaultMessage="Host Isolation Exceptions"
+        />
+      }
+      actions={[]}
+    >
+      <SearchExceptions
+        defaultValue={location.filter}
+        onSearch={handleOnSearch}
+        placeholder={i18n.translate(
+          'xpack.securitySolution.hostIsolationExceptions.search.placeholder',
+          {
+            defaultMessage: 'Search on the fields below: name, description, ip',
+          }
+        )}
+      />
+      <EuiSpacer size="m" />
```

Comment:
Why do you need two EuiSpacer here? If you need more space, you can just use the size="l" instead

Reply:
This is copy pasted code. But you are right, I'll replace it for an `l`